src/data/markdown/translated-guides/en/02 Using k6/02 Metrics.md

```diff
@@ -69,16 +69,80 @@ All the `http_req_...` lines and the ones after them are _built-in_ metrics that
 
 The following _built-in_ metrics will **always** be collected by k6:
 
-| Metric Name        | Type    | Description                                                                                                                                                                                                     |
-| ------------------ | ------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| vus                | Gauge   | Current number of active virtual users                                                                                                                                                                          |
-| vus_max            | Gauge   | Max possible number of virtual users (VU resources are pre-allocated, to ensure performance will not be affected when scaling up the load level)                                                                |
-| iterations         | Counter | The aggregate number of times the VUs in the test have executed the JS script (the `default` function).                                                                                                         |
-| iteration_duration | Trend   | The time it took to complete one full iteration of the default/main function.                                                                                                                                   |
+| Metric Name | Type | Description |
+| ----------- | ---- | ----------- |
+| vus                | Gauge   | Current number of active virtual users |
+| vus_max            | Gauge   | Max possible number of virtual users (VU resources are pre-allocated, to ensure performance will not be affected when scaling up the load level) |
+| iterations         | Counter | The aggregate number of times the VUs in the test have executed the JS script (the `default` function). |
+| iteration_duration | Trend   | The time it took to complete one full iteration. It includes the time spent in `setup` and `teardown` as well, there is currently a workaround for excluding them shown in the example below. |
```

Comment:
```suggestion
| iteration_duration | Trend   | The time it took to complete one full iteration. It includes the time spent in `setup` and `teardown` as well. To exclude them, see the workaround in the example below. |
```

Reply:
I would argue this is ... wrongly worded

`itearation_duration{scenario:default}` *does* not actually *exclude* `setup` and `teardown` times. It looks like it, but what it actually does is make a new *submetric* (w/e that means) that is all values of `iteration_duraiton` that have a tag `scanario` with value `default` - theoretically at least (especially with `k6/execution`) you can have metrics with that tag and value that are emitted in `setup` and also you can have another 20 scenarios with different names and this will "exclude" all of them

Reply:
```suggestion
| iteration_duration | Trend   | The time it took to complete one full iteration. It includes the time spent in `setup` and `teardown` as well. See the workaround in the example below, how to create a sub-metric for getting only the duration of the iteration's function for a specific scenario.  |
```
Could be better in this way?